frontends/web/src/api/backend.ts

```diff
@@ -30,6 +30,11 @@ export interface ISuccess {
     errorCode?: string;
 }
 
+export type AddrSignResp = {
```

Comment:
nit: we usually spell out all words, `AddressSignResponse`

Reply:
ACK